cdk/lib/super-mode-calculator.ts

```diff
@@ -0,0 +1,97 @@
+import {GuScheduledLambda} from "@guardian/cdk";
+import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
+import {GuStack, GuStringParameter} from '@guardian/cdk/lib/constructs/core';
+import {type App, Duration, RemovalPolicy} from 'aws-cdk-lib';
+import {AttributeType, BillingMode, Table} from "aws-cdk-lib/aws-dynamodb";
+import { Schedule } from 'aws-cdk-lib/aws-events';
+import { PolicyStatement, Role, ServicePrincipal} from 'aws-cdk-lib/aws-iam';
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+
+const appName = 'super-mode-calculator-app';
+export class SuperModeCalculator extends GuStack {
+	constructor(scope: App, id: string, props: GuStackProps) {
+		super(scope, id, props);
+
+		const scheduleState = new GuStringParameter(this, 'ScheduleState', {
+			description: 'The state of the schedule',
+			default: 'ENABLED',
+			allowedValues: ['ENABLED', 'DISABLED'],
+		});
+
+		const scheduleRules =
+			scheduleState.valueAsString === 'ENABLED'
+				? [
+					{
+						schedule: Schedule.rate(Duration.minutes(60)),
+					},
+				]
+				: [];
+
+		const superModeCalculatorTable = new Table(this, 'super-mode-calculator-table', {
+			tableName: `super-mode-${this.stage}`,
```

Comment:
I think this will clash with the existing table name. `super-mode-v2-${this.stage}`?